tests/Feature/Sensors/ScheduledTaskSensorTest.php

```diff
@@ -163,27 +166,26 @@
 });
 
 it('resets trace ID and timestamp on each task run', function () {
-    Schedule::call(fn () => travelTo(now()->addMicroseconds(1_000_000)))->everyMinute();
+    app(Schedule::class)->call(fn () => travelTo(now()->addMicroseconds(1_000_000)))->everyMinute();
 
     Str::createUuidsUsing(fn () => '00000000-0000-0000-0000-000000000001');
     Artisan::call('schedule:run');
 
     $this->ingest->assertWrittenTimes(1);
     $this->ingest->assertLatestWrite('scheduled-task:0.trace_id', '00000000-0000-0000-0000-000000000001');
     $this->ingest->assertLatestWrite('scheduled-task:0.timestamp', 946688523.456789);
-    $this->ingest->flush();
 
     Str::createUuidsUsing(fn () => '00000000-0000-0000-0000-000000000002');
     Artisan::call('schedule:run');
 
-    $this->ingest->assertWrittenTimes(1);
+    $this->ingest->assertWrittenTimes(2);
```

Comment:
Reading this test, I really wanted to see that is had written another time, so I removed the manual flush.